### PR TITLE
test(app): fix EBUSY/EPERM errors when running e2e tests on Windows

### PR DESCRIPTION
**Pull request by flotwig**
- Closes UNIFY-890

### User facing changelog

n/a

### Additional details

- Fixes EBUSY/EPERM errors when running app e2e tests on Windows.
- EBUSY is caused by AV/NTFS holding a lock on recently created files, so when we try to delete them, it fails. Ignoring the error when it happens on Windows bypasses this, and should be relatively safe since the project will be rescaffolded anyways before the next test.
- EPERM seems to be caused by `winston` corrupting file permissions when writing to `all.log`. The fix for this is removing the winston logging, see 8705be58af73da675efa7ddbec7d11c324d35f93 for further rationale on removing winston.
- `run-launchpad-integration-tests-chrome` is also failing in `10.0-release`: https://app.circleci.com/pipelines/github/cypress-io/cypress/30254/workflows/b9aec1ef-b00c-476f-8f65-bd1300669638/jobs/1172198
- Fixes an issue where `globby` did not work on Windows making supportFile not found: 525c82d
- Fixes wonky `--project` parsing in Gulp scripts (thanks, Bart): 9aa546a 

### PR Tasks

- [na] Have tests been added/updated?
	- Running the app tests on Windows will be added in a future PR.
- [na] Has the original issue (or this PR, if no issue exists) been tagged with a release in ZenHub? (user-facing changes only)
- [na] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [na] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
- [na] Have new configuration options been added to the [`cypress.schema.json`](https://github.com/cypress-io/cypress/blob/develop/cli/schema/cypress.schema.json)?

### CODE REVIEW

File: system-tests/lib/fixtures.ts [hunk 1]
```diff
@@ -11,6 +11,21 @@ const serverRoot = _path.join(__dirname, '../../packages/server/')
 const projects = _path.join(root, 'projects')
 const cyTmpDir = _path.join(tempDir, 'cy-projects')
 
+const safeRemove = (path) => {
+  try {
+    fs.removeSync(path)
```
Comment (ryanthemanuel) [inline system-tests/lib/fixtures.ts:16]: Can we not keep this async? i.e. make safeRemove async and change this to await fs.remove?

**Review by ryanthemanuel — DISMISSED** [review]

File: system-tests/lib/fixtures.ts [hunk 2]
```diff
@@ -11,6 +11,21 @@ const serverRoot = _path.join(__dirname, '../../packages/server/')
 const projects = _path.join(root, 'projects')
 const cyTmpDir = _path.join(tempDir, 'cy-projects')
 
+const safeRemove = (path) => {
```
Comment (elevatebart) [inline system-tests/lib/fixtures.ts:14]: Could this leave some files in the file system after running all tests if it happens at the wrong moment?
Maybe we could do a complete cleanup at the end of the test cycle with a short delay to make sure we have it all cleaned up.

**Review by elevatebart — DISMISSED** [review]
The code looks good, the explanation makes sense.
Could we make sure the CI runs on windows on this branch before merging?

I'll update the `circle.yml`

**Comment by elevatebart** [comment]
On my machine, it systematically times out. @flotwig How did you manage to run your tests?

File: system-tests/lib/fixtures.ts (hunk 2 above)
Comment (flotwig) [reply system-tests/lib/fixtures.ts:14]: Maybe, but I don't think it's necessary to worry about, it all goes into your tmp dir anyways.

**Comment by flotwig** [comment]
> On my machine, it systematically times out. @flotwig How did you manage to run your tests?

@elevatebart this does not fix all of the issues with running the app e2e tests, just the EPERM/EBUSY errors. I ran in `open` mode. The tests should run but still fail. But I am now getting a different error locally after merging 10.0-release :sweat: I'll put this in draft

File: packages/server/lib/exception.js [hunk 3]
```diff
@@ -61,5 +56,8 @@ module.exports = {
     .spread((body, authToken) => {
```
[the remark this answers is not in the document]
Comment (flotwig) [reply packages/server/lib/exception.js]: 695788f093acfd9e7189fd209d8db42bcb70845b

**Review by elevatebart — APPROVED** [review]

**Review by ryanthemanuel — APPROVED** [review]

**Pull request merged**